src/index.html

```diff
@@ -11,7 +11,162 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+
+      <fieldset>
+        <legend>Personal Information</legend>
+
+        <label class="pers-inf">
+          Surname:
+          <input
+          type="text"
+          name="surname"
+          required
+          autocomplete="off"
+          >
+        </label>
+
+        <label class="pers-inf">
```

Comment:
```suggestion
        <label class="person-info">
```
do not cut class names